python/tvm/contrib/popen_pool.py

```diff
@@ -269,21 +285,32 @@ class PopenPoolExecutor:
 
     timeout : float
         Timeout value for each function submit.
-    Note
```

Comment:
Keep this line

Reply:
Thanks @junrushao1994 for adding this line and @vinx13 for the comments!